Source/Core/Core/Core.cpp

```diff
@@ -858,23 +860,35 @@ void VideoThrottle()
 
 // --- Callbacks for backends / engine ---
 
-// Should be called from GPU thread when a frame is drawn
+// Called from Renderer::Swap (GPU thread) when a frame is presented to the user's screen
 void Callback_VideoCopiedToXFB(bool video_update)
```

Comment:
This function is now seriously misnamed. 

Can we get it updated to something like `Callback_FrameDrawnToHostScreen` or `Callback_FramePresented`

And expand the explanatory comment to say it filters duplicates.

Reply:
Also, should we be deleting the video_update argument.

We are always calling this function with video_update = true.

Reply:
Done (in a separate commit, because the cause for this needing to be renamed is not this PR).

For the reference, further in the past we used to call this function with false for duplicate frames, but there is no reason to do that anymore since there now is no code in this function that runs when `video_update` is false.